.irb_config

```diff
@@ -0,0 +1,11 @@
+# Not that we use a non-standard config file name to reduce
```

Comment:
How about `.brew_irbc` or similar? Similarly, putting this under `Library/Homebrew` as a not dot file will make it a bit more discoverable.

Reply:
I thought it made sense to make it a dot file since it's just a config thing but I have no preference here.

Reply:
Similarly using `brew_irbrc` makes clearer it's for `brew irb`

Reply:
Addressed in https://github.com/Homebrew/brew/pull/14892/commits/deabd4a5f9adb3fa51ac732c86610c700b52e96d